### ISSUE TITLE
Crash Report after updating server to 0.3.9 (release)

### ISSUE THREAD

**Issue by Stonewolfe** [NillerMedDild/Enigmatica6#1087]
**Modpack version** 
0.3.9 (release)

**Describe the bug**
The server delivers crash report upon trying to start

**Logs**
https://pastebin.com/rjVGckG9

I am not brilliant with this things so I am not sure what is going on here and I am illiterate when reading these things. Why is my server refusing to start after the update?

[crash-2021-01-26_22.48.00-server.txt](https://github.com/NillerMedDild/Enigmatica6/files/5876685/crash-2021-01-26_22.48.00-server.txt)

**Comment by NillerMedDild** [comment]
Hmm, it looks like some atuocrafting with Refined Storage is going wrong.
@Darkere do you know anything about this?

**Comment by NillerMedDild** [comment]
I doubt it will fix it, but you can try updating to 0.3.10 for now.

**Comment by Stonewolfe** [comment]
@NillerMedDild interesting as I know no one on the server has made it to refined storage yet... I will have to wait to see if Akliz will put 3.10 up... I also tried rolling back and it gave me the same error... though that might have been an issue on my side I am not sure.

**Comment by Darkere** [comment]
A Craftingtask is missing data. Soo... I'm pretty sure someone has made it to Refinedstorage.

This error as far as I know only happens when upgrading from an older version of RS?

Open up world/data/refinedstorage_networks.dat and find and delete the crafting task. 

Depending on where the network is, it's possible that it is in the data folder of another dimension.

**Comment by Stonewolfe** [comment]
# @Darkere that is really odd for us only because we haven't even opened a nether portal yet. It's just 3 of us who play together on Mondays for streams.

Sorry for being slow here probably but what is nbt explorer? Is that similar to the Filezilla thing Akliz recommended to look at server files?

**Comment by Merlo17** [comment]
Not really, it's a tool used to open and edit minecraft .dat nbt files

https://github.com/jaquadro/NBTExplorer/releases

**Comment by Stonewolfe** [comment]
So I am not sure why... and I am definitely not sure how... but I think my server was holding onto an old world from before I think this pack even had quests when we had played for a bit before choosing a different pack to play while the quests were released and developed. I think it was that world that was causing issues as I went through my files and noticed two world folders. One with refined storage data and the other without. I deleted the one with the refined storage data completely and it loaded up this time luckily still the world we have been playing it for the last two weeks.

So thank you @Darkere for leading me in the direction of the data. I thought it was odd because we haven't even put enough out in the terms of power to run refined storage because we have all been focusing on the Create mod. I think one of us did a little in Immersive, enough for a windmill and water wheel. But otherwise its mostly been Create and... usual vanilla-esque living (crops/cows/sheep/etc).
